sparkmagic/sparkmagic/tests/test_sqlquery.py

```diff
@@ -61,26 +61,26 @@ def test_pyspark_livy_sql_options():
     query = "abc"
 
     sqlquery = SQLQuery(query, samplemethod='take', maxrows=120)
-    assert_equals(sqlquery._pyspark_command(),
+    assert_equals(sqlquery._pyspark_command("sqlContext"),
```

Comment:
should we use 'spark' as the default for these tests? That would be more future proof I guess?